    @Test
    public void testWrappedFor_2_Bug277625() throws Exception {
        assertIndenterResult(
                "for (int i = 0; i < 2;\n" +
                "i++)",

                "for (int i = 0; i < 2;\n" +
                "\t\ti++)");
    }

